innopoints/views/project.py

```diff
@@ -74,6 +62,14 @@ def list_projects():
         if order_by == 'proximity':
             db_query = db_query.join(Activity).group_by(Project.id)
         db_query = db_query.order_by(ordering[order_by, order])
+    elif request.args.get('type') == 'past':
```

Comment:
So we are not using 'finalizing' and 'finished' in the params? Since the project is sent with these values for the lifetime_stage, it seems inconsistent that the request uses a different format.
Which also bring to attention, doesn't "stage" seem more appropriate than "type" for the param name?

Reply:
That's the thing. `type` was used instead of `stage` because they are not the same. For the user interface it's valid to ask for all past projects, and the backend decides which projects are past.